src/app/config/schema.ts

```diff
@@ -294,19 +294,19 @@ export const optionalVarsSchema: Schema<IOptionalVarsSchema> = {
     respondentRolloutEmail: {
       doc: 'Percentage threshold to serve React for respondents for Phase 1 (email mode forms)',
       format: 'int',
-      default: 0,
+      default: 1,
```

Comment:
Why are we changing these to 1 again?

Reply:
this was my mistake - i meant to change `docker-compose.yml` instead so that the banner message shows up.  Changed this back to `0`!